modules/android/partials/release-notes/couchbase-mobile-android-release-note.3.3.0.adoc

```diff
@@ -14,6 +14,8 @@ Version 3.3.0 for {param-title} delivers the following features and enhancements
 * https://jira.issues.couchbase.com/browse/CBL-7214[CBL-7214 -- Update minimum supported API level to 24]
 * https://jira.issues.couchbase.com/browse/CBL-7225[CBL-7225 -- Add info about default collection used in the replication]
 * https://jira.issues.couchbase.com/browse/CBL-7233[CBL-7233 -- Add API for creating `ReplicatorConfiguration` with collection configs and endpoint]
+* https://jira.issues.couchbase.com/browse/CBL-7360[CBL-7360 -- Update mbedTLS to 3.6.4]
+* https://jira.issues.couchbase.com/browse/CBL-7409[CBL-7409 -- Improve conflict resolution with multiple remotes]
```

Comment:
Sorry, I just saw that the CBL-7194-Upgrade mbedTLS to 3.6.3 should be removed as we updated it to 3.6.4 instead. I also update my release note report.

Reply:
I've updated it now